File: sdk/android/component/sqljson/controller/FileController.java
package ardvro.component.sqljson.controller;

import ardvro.core.connection.IConnection;
import ardvro.core.connection.IResponseListener;

public class FileController
{
    private IConnection _conection;

    public FileController(IConnection connection)
    {
        _conection = connection;
    }

    public void Get(Object id, IResponseListener<byte[]> onResponse)
    {
        String functionName = "ardvro/component/sqljson/File/Get";
        _conection.Submit(new Object[]{ id }, functionName, onResponse);
    }

}

File: sdk/web/component/sqljson/controller/FileController.js
var FileController = function FileController(cfg)
{
    let ctrl = {};

    ctrl.Get = function (id, onResponse)
    {
        let args = [id];
        cfg.Connector.Submit(args, "ardvro/component/sqljson/File/Get", function (box)
        {
            onResponse != null ? onResponse(box == null ? null : box.Data) : null;
        });
    };

    

    return ctrl;
};